test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -2518,6 +2518,33 @@ var contentTypesReader
             }
         }
 
+        [Fact]
+        public void PackageBuilderWorksWithFilesHavingCurrentDirectoryAsTarget()
```

Comment:
Can we test other similar cases? e.g. `./blah/../foo.txt`, `.\blah\..\foo.txt`, `..\foo.txt`, `foo\.\.\.\bar\..\bar.txt`.